contracts/Vault.vy

```diff
@@ -142,57 +203,162 @@ def setSymbol(_symbol: String[20]):
 # 2-phase commit for a change in governance
 @external
 def setGovernance(_governance: address):
+    """
+    @notice
+        Nominate a new address to use as governance.
+
+        The change does not go into effect immediately. This function sets a
+        pending change, and the governance address is not updated until
+        the proposed governance address has accepted the responsibility.
+
+        This may only be called by the current governance address.
+    @param _governance The address requested to take over Vault governance.
+    """
     assert msg.sender == self.governance
     self.pendingGovernance = _governance
 
 
 @external
 def acceptGovernance():
+    """
+    @notice
+        Once a new governance address has been proposed using setGovernance(),
+        this function may be called by the proposed address to accept the
+        responsibility of taking over governance for this contract.
+
+        This may only be called by the proposed governance address.
+    @dev
+        setGovernance() should be called by the existing governance address,
+        prior to calling this function.
+    """
     assert msg.sender == self.pendingGovernance
     self.governance = msg.sender
 
 
 @external
 def setRewards(_rewards: address):
+    """
+    @notice
+        Changes the rewards address. Any distributed rewards
+        will cease flowing to the old address and begin flowing
+        to this address once the change is in effect.
+
+        This will not change any Strategy reports in progress, only
+        new reports made after this change goes into effect.
+
+        This may only be called by governance.
+    @param _rewards The address to use for collecting rewards.
+    """
     assert msg.sender == self.governance
     self.rewards = _rewards
 
 
 @external
 def setDepositLimit(_limit: uint256):
+    """
+    @notice
+        Changes the maximum amount of tokens a single address may have
```

Comment:
this may confuse the readers, reads like a cap for a single address where its in fact a cap for total deposits hence why NOTE is needed in line 263. 

`Changes the maximum amount of tokens that can be deposited in this Vault.``

Reply:
Ah good point. I'll fix it now.